hie-plugin-api/hie-plugin-api.cabal

```diff
@@ -49,10 +49,10 @@ library
                      , fingertree
                      , free
                      , ghc
-                     , hie-bios
+                     , hie-bios == 0.3.*
                      , ghc-project-types >= 5.9.0.0
                      , cabal-helper
-                     , haskell-lsp == 0.18.*
+                     , haskell-lsp == 0.19.*
                      , hslogger
                      , unliftio
                      , monad-control
```

Comment:
Can you also remove `rope-utf16-splay >= 0.3.1.0` from this too? I don't think its needed anymore